apps/script/aichatCloudWatchDashboard/constants.js

```diff
@@ -0,0 +1,217 @@
+const DASHBOARD_NAME = 'GenAI';
+
+const modelNames = [
```

Comment:
Can we pull these from `static/aichat/modelDescriptions.json`? and/or `sharedConstants.ts#AiChatModelIds`?

Reply:
Yep, done!